lib/crc8.c

```diff
@@ -24,7 +24,7 @@
 
 #ifdef __linux__
 
-#include "crc8.h"
```

Comment:
Better to do a file rename in a different commit.

Reply:
Updated, please review.